docs/pages/modules/get-started.mdx

```diff
@@ -44,19 +44,10 @@ If you have an **ios** directory in your project that you created using `npx exp
 > **Note:**
 > If you're using a development client, you need to rebuild your development client any time you want to use new native code.
 
-> **Warning:**
-> If you override the `paths` property in `tsconfig.json` or are using Expo SDK version below `48.0.11`, you need to include the following wildcard in [the paths config](https://www.typescriptlang.org/tsconfig#paths).
->
-> ```
-> "paths": {
->   "*": ["./modules/*"]
-> }
-> ```
-
-Now, import the module in your application:
+Now, import the module in your application, for example in `App.js` or `App.tsx`:
```

Comment:
We could have made filenames bold instead of using inline code syntax as [per our writing styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text). Will create a follow-up PR (just a heads up) to apply that 😄